resources/shaders/dxr_reflection_functions.hlsl

```diff
@@ -85,8 +85,9 @@ float4 DoReflection(float3 wpos, float3 V, float3 N, uint rand_seed, uint depth,
 
 		// Shoot perfect mirror ray if enabled or if it's a recursion or it's almost a perfect mirror
 
-		if (depth > 0 || roughness < 0.05f)
+		if (depth > 0 || roughness < 0.075f)
 			return float4(TraceReflectionRay(wpos, N, reflected, rand_seed, depth, cone, dir_t), 1);
+	#endif
```

Comment:
This means that turning on PERFECT_MIRROR_REFLECTIONS will instead give you rough reflections, including for recursions, which means the define now does the opposite of what it should do.